modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/ProjectErrorMessage/ProjectErrorMessage.jsx

```diff
@@ -0,0 +1,36 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import ClayIcon from '@clayui/icon';
+import i18n from '~/common/I18n';
+
+const ProjectErrorMessage = () => {
+	return (
+		<div className="align-items-center cp-project-error-message-page d-flex flex-column justify-content-center">
+			<div className="cp-project-error-message-container">
+				<div className="align-items-center cp-project-error-message-alert d-flex flex-column mt-7">
+					<div className="align-items-center cp-project-error-message-card-icon d-flex justify-content-center">
+						<ClayIcon
+							className="cp-project-error-message-icon text-primary"
+							symbol="lock"
+						/>
+					</div>
+
+					<p className="cp-project-error-message-title mt-4">
+						{i18n.translate('you-dont-have-access-to-this-project')}
+					</p>
+				</div>
+
+				<p className="cp-project-error-message-informative mt-2 mx-9 text-center">
+					{i18n.translate(
+						'make-sure-the-project-link-is-correct-and-that-you-have-access-to-this-project'
+					)}
+				</p>
+			</div>
+		</div>
+	);
+};
```

Comment:
As there is no logic in this component, I generally like to return the component directly (without the "return" statement)

```suggestion
const ProjectErrorMessage = () => (
		<div className="align-items-center cp-project-error-message-page d-flex flex-column justify-content-center">
			<div className="cp-project-error-message-container">
				<div className="align-items-center cp-project-error-message-alert d-flex flex-column mt-7">
					<div className="align-items-center cp-project-error-message-card-icon d-flex justify-content-center">
						<ClayIcon
							className="cp-project-error-message-icon text-primary"
							symbol="lock"
						/>
					</div>

					<p className="cp-project-error-message-title mt-4">
						{i18n.translate('you-dont-have-access-to-this-project')}
					</p>
				</div>

				<p className="cp-project-error-message-informative mt-2 mx-9 text-center">
					{i18n.translate(
						'make-sure-the-project-link-is-correct-and-that-you-have-access-to-this-project'
					)}
				</p>
			</div>
		</div>
	);
```

Reply:
Uh, you need a return or the page errors out and won't render. Is there a different format you want me to use instead?

Reply:
No, it's actually just a syntax pattern. You can remove the "return" and brackets, like I suggested above.
`const ProjectErrorMessage = () => (<div></div>)`

Reply:
thanks Jair, I removed the return but didn't change the brackets, works now 🙇